test_conformance/basic/test_enqueue_map.cpp

```diff
@@ -197,9 +202,11 @@ int test_enqueue_map_image(cl_device_id deviceID, cl_context context, cl_command
           mappedPtr[ ( y * rowPitch/sizeof(cl_uint) ) + x ] = value;
 
           // Also update the initial data array
-          value = referenceData[ ( ( offset[ 1 ] + y ) * imageSize + offset[ 0 ] ) * 4 + x ];
+          value =
```

Comment:
I definitely don't want to start formatting wars, but this section gets harder to read with the new format. I'm not sure if it's the splitting at the =, the max line length, or the combination of the two.

Reply:
@jlewis-austin could you put any formatting discussion points in #625 ? That way we can look at them all in one place.

Reply:
Sorry I missed this before merging. I agree the formatting doesn't look great there. Indenting all but the first line of the expressions would make for more readable code. I'm sure there's an option for this.

Reply:
@Jeremy-Kemp done

Reply:
This is probably fixed by #739